carma_wm_ctrl/include/carma_wm_ctrl/GeofenceSchedule.hpp

```diff
@@ -67,7 +68,7 @@ class GeofenceSchedule
    *
    * @return True if time > schedule_end
    */
-  bool scheduleExpired(const ros::Time& time = ros::Time::now()) const;
+  bool scheduleExpired(const rclcpp::Time& time = rclcpp::Time(0.0,0.0,RCL_SYSTEM_TIME)) const;
```

Comment:
Can you confirm this is a safe change. Is this called anywhere with the default arguments?

Reply:
I think that's fine